internal/work/dispatcher.go

```diff
@@ -179,8 +186,89 @@ func (d *Dispatcher) Connect() error {
 						log.Errorf("cannot add journal entry: %v", err)
 					}
 				}()
+			case "org.freedesktop.DBus.NameOwnerChanged":
+				name, ok := s.Body[0].(string)
+				if !ok {
+					log.Errorf("cannot convert body element 0 (name) to string")
```

Comment:
These should be `log.Error` instead of `log.Errorf` similar to this commit e4811b7

Reply:
Good catch. Thank you.